tests/runner/src/env.rs

```diff
@@ -308,14 +310,14 @@ impl Env {
     /// Build the DB with `cargo build --bin greptime`
     async fn build_db() {
         println!("Going to build the DB...");
-        let cargo_build_result = Command::new("cargo")
+        let output = Command::new("cargo")
             .current_dir(util::get_workspace_root())
             .args(["build", "--bin", "greptime"])
-            .stdout(Stdio::null())
             .output()
-            .expect("Failed to start GreptimeDB")
-            .status;
-        if !cargo_build_result.success() {
+            .expect("Failed to start GreptimeDB");
+        if !output.status.success() {
+            io::stdout().write_all(&output.stdout).unwrap();
+            io::stderr().write_all(&output.stderr).unwrap();
```

Comment:
Print some separator or opening word for the incoming error

Reply:
@waynexia PTAL